po/vi.po

```diff
@@ -2548,52 +2552,56 @@ msgstr ""
 
 #: src/hello-world/what-is-rust.md
 msgid "Rust is used for a wide range of devices:"
-msgstr ""
+msgstr "Rust được sử dụng cho nhiều loại thiết bị:"
 
 #: src/hello-world/what-is-rust.md
 msgid "firmware and boot loaders,"
-msgstr ""
+msgstr "firmware và các chương trình khởi động (*boot loaders*),"
 
 #: src/hello-world/what-is-rust.md
 msgid "smart displays,"
-msgstr ""
+msgstr "màn hình thông minh,"
 
 #: src/hello-world/what-is-rust.md
 msgid "mobile phones,"
-msgstr ""
+msgstr "điện thoại di động,"
 
 #: src/hello-world/what-is-rust.md
 msgid "desktops,"
-msgstr ""
+msgstr "máy tính,"
 
 #: src/hello-world/what-is-rust.md
 msgid "servers."
-msgstr ""
+msgstr "máy chủ."
 
 #: src/hello-world/what-is-rust.md
 msgid "Rust fits in the same area as C++:"
-msgstr ""
+msgstr "Rust phù hợp trong cùng lĩnh vực như C++:"
 
 #: src/hello-world/what-is-rust.md
 msgid "High flexibility."
-msgstr ""
+msgstr "Độ linh hoạt cao."
 
 #: src/hello-world/what-is-rust.md
 msgid "High level of control."
-msgstr ""
+msgstr "Có quyền điều khiển cao."
 
 #: src/hello-world/what-is-rust.md
 msgid ""
 "Can be scaled down to very constrained devices such as microcontrollers."
 msgstr ""
+"Có thể ứng dụng với các thiết bị có tài nguyên hạn chế như vi điều khiển."
 
 #: src/hello-world/what-is-rust.md
 msgid "Has no runtime or garbage collection."
 msgstr ""
+"Không có thời điểm chạy (*runtime*) hay trình thu dọn rác (*garbage "
```

Comment:
Hmm, I suppose "thời gian chạy" is a better trans? "Thời điểm" sounds a bit vague? But generally this already looks good enough for a merge.

Reply:
Agree! I was also confused about that terminology. I think there is no clear translation of that terminology. Thanks for your suggestion!